spec/requests/internal/feedback_messages_spec.rb

```diff
@@ -15,6 +16,16 @@
         expect(response.status).to eq 200
       end
     end
+
+    context "when there is a vomit reaction on a user" do
+      it "renders with status 200" do
+        trusted_user
+        create(:reaction, category: "vomit", reactable: user, user: trusted_user)
+        sign_in admin
+        get internal_reports_path
+        expect(response.status).to eq 200
+      end
```

Comment:
This was incredibly slow locally, and I'm not sure why... It took 2 min 18 seconds to complete this one test. Hopefully doesn't add too much time to our CI. 😬